### PR TITLE
Spark cassandra api

### LINKED ISSUE #7: Resolve problem with cassandra tables
There is a strange issue with CassandraClient and cassandra driver. We create a table with uppercase characters in the name and its lowercased later. Some queries are forced to use the lowercase version, some of them not. We should at least hide the .toLowerCase method invocation from the user of the client.

### PR DESCRIPTION
**Pull request by Adebski**
Closes #7 and #8 and partially #22

### CODE REVIEW

File: src/main/scala/ztis/KafkaComponent.scala [hunk 1]
```diff
@@ -7,15 +7,13 @@ import com.twitter.chill.{KryoInstantiator, KryoPool}
...
-import ztis.wykop.WykopKryoRegistrar
 
 import scala.collection.JavaConverters._
 
 
 trait KafkaComponent extends StrictLogging {
-  val kryo = KryoPool.withByteArrayOutputStream(1, (new KryoInstantiator)
-    .withRegistrar(new TweetKryoRegistrar).withRegistrar(new WykopKryoRegistrar))
+  private val kryoInstantiator = (new KryoInstantiator).withRegistrar(new ZTISKryoRegistrar)
+  val kryo = KryoPool.withByteArrayOutputStream(1, kryoInstantiator)
```
Comment (Mequrel) [inline src/main/scala/ztis/KafkaComponent.scala]: add named arguments to explicitly give meaning to "1"

File: src/main/resources/cassandra.conf [hunk 2]
```diff
@@ -2,4 +2,6 @@ cassandra {
   contact-points = ["127.0.0.1:9042"]
   keyspace = "recommender"
   ratings-table-name = "ratings"
+  user-features-table-name = "modelUserFeatures"
+  product-features-table-name = "modelProductFeatures"
```
Comment (Mequrel) [inline src/main/resources/cassandra.conf]: Why have you moved that?

File: src/main/scala/ztis/cassandra/CassandraClient.scala [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+import com.datastax.driver.core.{Cluster, ResultSet}
+import com.typesafe.config.Config
+import com.typesafe.scalalogging.slf4j.StrictLogging
+import ztis.UserAndRating
+
+import scala.collection.JavaConverters._
+
+class CassandraClient(config: Config) extends StrictLogging {
+
+  private[cassandra] val keyspace: String = CassandraClient.keyspace(config)
```
Comment (Mequrel) [inline src/main/scala/ztis/cassandra/CassandraClient.scala]: I would extract configuration case class. We are mixing configuration parsing and simple CQL actions here.

**Comment by Mequrel** [comment]
Added minor comments. Really great and useful job! LGTM

File: src/main/resources/cassandra.conf (hunk 2 above)
Comment (Adebski) [inline src/main/resources/cassandra.conf]: It is easier to have all the configuration keys in one place. If there would be situations when one component load CassandraClient class but does not have those keys in the loaded configuration files the logic would be much more complicated. We would have to use Option types or hardcode some default values inside Scala files. 

I decided it is easier to just move all table name configuration inside single file.

File: src/main/scala/ztis/KafkaComponent.scala (hunk 1 above)
Comment (Adebski) [inline src/main/scala/ztis/KafkaComponent.scala]: KryoPool is a class from twitter-chill library: https://github.com/twitter/chill and this class is written in Java so there is no access to named arguments. I will extract the 1 value to private val and give it some meaningful name.

**Pull request merged**